docs/tools/SvPileup.md

```diff
@@ -9,12 +9,47 @@ title: SvPileup
 
 Collates a pileup of putative structural variant supporting reads.
 
+## Outputs
+
 Two output files will be created:
 
-1. `<output-prefix>.txt`: a tab-delimited file describing SV pileups, one line per breakpiont event.
+1. `<output-prefix>.txt`: a tab-delimited file describing SV pileups, one line per breakpoint event.  The returned
+   breakpoint will be canonicalized such that the "left" side of the breakpoint will have the lower (or equal to)
+   position on the genome vs. the "right"s side.
 2. `<output-prefix>.bam`: a SAM/BAM file containing reads that contain SV breakpoint evidence annotated with SAM
-  tags.  The `ev` SAM tag lists the type of evidence found, while the `be` list the unique breakpoint identifier
-  output in (1) above.
+  tag.
+
+The `be` SAM tag contains a comma-delimited list of breakpoints to which a given read belongs.  Each element is
+a semi-colon delimited, with four fields:
```

Comment:
Maybe give an example of a read that supports two breakends (is that really a thing?) just to make it clear how `,` and `;` are used.